styles/buttonSectionStyles.css

```diff
@@ -31,7 +31,6 @@
     justify-content: space-between;
     margin: 0 auto;
     max-width: 480px;
-    padding: 0 6px;
     width: 100%;
```

Comment:
width:100% raczej jest zbędne,ale przetestój dla pewności

Reply:
racja, to nie tutaj było potrzebne :d wyrzucam